examples/Modals/ModalHeader.md

```diff
@@ -77,3 +77,85 @@ const [modalVisible, setModalVisible] = useState(false);
   </Modal>
 </div>
 ```
+
+### With the `forwardRef`
+Ref example: The optional `forwardRef` property is set. When the test button is
+clicked, the focus shifts to the modal header.
+```jsx
+import { useState, useRef, } from 'react';
+import { VARIANT } from 'Theme';
+import {
+  Button,
+  Dropdown,
+  Modal,
+  ModalBody,
+} from 'mark-one';
+
+const RefExample = () => {
+  const ref = useRef(null);
+  const [value, setValue] = useState('');
+  const [modalVisible, setModalVisible] = useState(false);
+  const onButtonClick = () => {
+    setModalVisible(true);
+    /* Since modal may not have been rendered in DOM, wait for it to be
+    rendered by letting next task of event queue run first */
+    setTimeout(() => ref.current.focus(), 0);
```

Comment:
for 0 timeout you can use ```setTimeout(() => ref.current.focus())```
Though I had bad experience with the setTimeout in general.

Reply:
Oh ok good to know. Yes I just needed a way to make sure the modal was rendered first.

Reply:
I updated this through 8c72c64.